buildlib/jucx-publish.yml

```diff
@@ -0,0 +1,49 @@
+parameters:
+  target: publish
```

Comment:
this means the default target is to publish a release? seems dangerous

Reply:
This file is a template. We it with parameters from both release and pipeline files.

Reply:
so can 'target' be empty string here in this file?